### ISSUE TITLE
Question about {{modifier_class}}

### ISSUE THREAD

**Issue by f1vlad** [kss-node/kss-node#516]
Hi, I have probably a quite obvious question, but maybe not, so I wanted to double check. Here's my sample doc:

```
// Base Button
//
// Markup:
// <ELEMENT class="button {{modifier_class}}"></ELEMENT>
//
// :active    - Active button
// :hover     - Highlight the button when hovered.
//
// Styleguide Forms.Button
```

**Result in DOM that I expect:**
```
<element class="button ">Label</element>
...
<element class="button pseudo-class-active">Label</element>
...
<element class="button pseudo-class-hover">Label</element>

```

**Question**

Do I expect KSS builder to automatically copy all pseudo classes like these:

.button:active {...} => .button.pseudo-class-active {...}
.button:hover {...} => .button.pseudo-class-hover {...}

for demo purposes?

Thanks.

**Comment by RobLoach** [comment]
I don't believe it'll create that CSS for you. It'll just write the markup. See the original PR for it over at:
https://github.com/kss-node/kss-node/pull/125

**Comment by f1vlad** [comment]
Actually they do get generated automatically for me and put in `<head>`

![Screenshot](https://i.imgur.com/V6wEKxj.png)

However, it only works when I run my [built] styleguide from real web server (http://path/to/styleguide) and not just load html files from file system.

**Comment by Epho** [comment]
Is it true that simluated psuedo-classes are only generated from a web server? I'm running from local file system and have been trying to figure out why they aren't working for me. I've noticed that the psuedo-classes are indeed created, I can see them in the DOM of my styleguide, but with no style definitions. The stylesheets don't seem to be generated for them.

![screen shot 2019-01-03 at 5 18 08 pm](https://user-images.githubusercontent.com/1441652/50669698-9b281480-0f7b-11e9-9186-db5354eef8a8.png)

**Comment by ryuran** [comment]
@Epho can you check if you have js error in your console
(I will try to reproduce as soon as possible)

**Comment by Epho** [comment]
@ryuran no I'm not getting any errors in my console.

**Comment by kasztof** [comment]
Is there any updated on this issue?
